packages/runtime/src/useCases/common/Schemas.ts

```diff
@@ -21972,6 +21975,10 @@ export const CreateOwnRelationshipTemplateRequest: any = {
             "type": "string",
             "errorMessage": "must match ISO8601 datetime format",
             "pattern": "^([+-]?\\d{4}(?!\\d{2}\\b))((-?)((0[1-9]|1[0-2])(\\3([12]\\d|0[1-9]|3[01]))?|W([0-4]\\d|5[0-2])(-?[1-7])?|(00[1-9]|0[1-9]\\d|[12]\\d{2}|3([0-5]\\d|6[1-6])))([T\\s]((([01]\\d|2[0-3])((:?)[0-5]\\d)?|24:?00)([.,]\\d+(?!:))?)?(\\17[0-5]\\d([.,]\\d+)?)?([zZ]|([+-])([01]\\d|2[0-3]):?([0-5]\\d)?)?)?)?$"
+        },
+        "AddressString": {
+            "type": "string",
+            "pattern": "did:e:[a-zA-Z0-9.-]+:dids:[0-9a-f]{22}"
```

Comment:
This regexp is in my opinion not the best for returning valid domain names, as it allowes `.....`, `----` or `a-.-` as examples.

Reply:
Do you have one in mind already? Before we go down the rabbithole of URL regexes

Also I don't think it's crucial to get this right - (hopefully) no one will manually enter an address and for copy-paste the check is not needed

Reply:
+1

Reply:
also - IF we want to change it it would definitely not be in this PR. This AddressString was introduced months before.

Reply:
Fixed in #294